tests/test_types_public_surface.py

```
@@ -0,0 +1,531 @@
+        assert roundtrip.params == original.params
+        assert roundtrip.no_human_mid_flow == original.no_human_mid_flow
+
+    def test_run_event_roundtrip(self) -> None:
+        """Test RunEvent serialization roundtrip with backwards compat."""
+        from swarm.runtime.types import (
+            RunEvent,
+            run_event_from_dict,
+            run_event_to_dict,
+        )
+
+        now = datetime.now(timezone.utc)
```

Ruff (0.14.10)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)